assets/dev/scss/frontend/widgets/tabs.scss

```diff
@@ -61,6 +61,10 @@
 					left: 100%;
 				}
 			}
+
+			&:focus {
+				text-decoration: underline;
+			}
```

Comment:
Do you say that the focus was not broken? it just was without UI? if yes I am not sure this is the solution.

Reply:
The answer is yes, @shilo-ey told its fine, what you suggest?